documentation/topics/ai/cursor-ai.md

```diff
@@ -0,0 +1,69 @@
+# Introduction
+
+This document contains tips and tricks for optimizing the experience when coding Elixir Phoenix Ash with CursorAI.
+
+## Models
+
+The model that generally provides the best performance as of this writing is Claude Sonnet 4. Thinking and MAX modes (which is significantly more expensive in terms of credits) tend to perform better depending on the prompt.
+
+## MCP Servers
+
+Follow this section to install the Tidewave, Ash-AI, and Playwright (optional) MCP servers.
+
+### Why Playwright?
```

Comment:
We can mention it, but I don't want to "bless" it in our docs. There are a lot of MCP vulnerabilities that come from agents reading things they shouldn't read, for example. https://invariantlabs.ai/blog/mcp-github-vulnerability